x-pack/plugins/ml/common/__mocks__/mock_anomalies_table_data.json

```diff
@@ -44,9 +45,11 @@
         "metricDescriptionSort": 82.83851409101328,
         "detector": "count by mlcategory",
         "isTimeSeriesViewDetector": false,
-          "influencers": [
-            "mockInfluencer"
-          ]
+        "influencers": [
+          {
+            "mockInfluencerField": "mockInfluencerValue"
+          }
+        ]
```

Comment:
Updating this to match the type `MlAnomaliesTableRecord`. Until now this mock was only used with `js` code, now also using it with `ts` code I noticed the structure was outdated for the `influencers` attribute.